ZappAppConnector/Protocols/ZAAppDelegateConnectorChromecastProtocol.h

```diff
@@ -19,7 +19,7 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)showExtededPlayer;
 - (nonnull UIViewController *)getExtededPlayerViewController;
 - (void)setCastDelegate:(nonnull id)castDelegate;
-
```

Comment:
Don't delete the empty line. White spacing = beauty :)

Reply:
Done. please approve.